docs/modules/tile-converter/cli-reference/tile-converter.md

```diff
@@ -76,9 +78,9 @@ NodeJS 14 or higher is required.
-| 7zExe                     | \*                        |                           | location of 7z.exe archiver to create slpk on Windows OS, default: "C:\\Program Files\\7-Zip\\7z.exe"                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
-| egm                       | \*                        | \*                        | location of the Earth Gravity Model (\*.pgm) file to convert heights from ellipsoidal to gravity-related format or "None" to not use it, default: "./deps/egm2008-5.pgm". A model file can be loaded from GeographicLib https://geographiclib.sourceforge.io/html/geoid.html                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| slpk                      | \*                        |                           | (Deprecated since version 4.3.0) <br> Whether the converter generates \*.slpk (Scene Layer Package) I3S output files. Note: For versions 4.3.0 and up \*.slpk is the default output wihtout this option specified.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
```suggestion
| slpk                      | \*                        |                           | (Deprecated since version 4.3.0) <br /> Whether the converter generates \*.slpk (Scene Layer Package) I3S output files. Note: For versions 4.3.0 and up the option doesn't work, \*.slpk is the only possible output of I3S.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Reply:
The option `--slpk` is removed from `npx tile-converter --help`. we should add this information back

Reply:
Where should it be brought back? I mean where is the help text located? Can you send file path here?

Reply:
Brought it back there.